arch/arm/core/aarch32/cortex_m/Kconfig

```diff
@@ -288,6 +288,13 @@ config SW_VECTOR_RELAY_CLIENT
 	  sure the vector table pointer in RAM is set properly by the image upon
 	  initialization.
 
+config CORTEX_M_DWT
+	bool "Enable and use the DWT"
+	depends on CPU_CORTEX_M_HAS_DWT
+	default y if TIMING_FUNCTIONS
```

Comment:
I see that now, maybe doing this the other way around, where we enable  CORTEX_M_DWT if TIMING_FUNCTIONS...

Reply:
Yes, good suggestion, will try that.

Reply:
fixed